src/plasoscaffolder/bll/services/base_sqlite_plugin_path_helper.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+import os
+from abc import ABCMeta, abstractmethod
+
+
+class BaseSqlitePluginPathHelper(metaclass=ABCMeta):
+  """Class representing the base class for the sqlite plugin path helper"""
+  def __init__(self, path: str, plugin_name: str):
+    """Initializes the sqlite plugin halper
+
+     Args:
+      path: the plaso folder path
+      plugin_name: The name of the plugin to check.
+
+    """
+
+  @abstractmethod
+  def formatter_file_path(self) -> str:
```

Comment:
FYI if these are just string values you could also implement these as instance attributes instead of instance methods.

```
def __init__(self, path: str, plugin_name: str):
  super(BaseSqlitePluginPathHelper, self).__init__()
  self.formatter_file_path = None
  self.parser_file_path = None
  ...
```